flipmart/flipmart/urls.py

```diff
@@ -18,15 +8,3 @@ def bad(request):
 ]
```

Comment:
path("i18n/", include("django.conf.urls.i18n")),
also remove this we don't need.